pmm-app/src/pmm-qan/panel/components/Details/database-models/mongodb/service.ts

```diff
@@ -1,7 +1,7 @@
-import { apiRequestManagement } from 'shared/components/helpers/api';
+import { apiRequestActions } from 'shared/components/helpers/api';
 
 export default {
   getTraditionalExplainJSONMongo(body) {
-    return apiRequestManagement.post<any, any>('/Actions/StartMongoDBExplain', body);
+    return apiRequestActions.post<any, any>(':startServiceAction', body);
```

Comment:
I haven't checked, but the body might need to be adjusted (seeing as the different StartTECHExplain merged into one).

Reply:
Updated.